Client/Frontend/Widgets/PhotonActionSheetProtocol.swift

```diff
@@ -121,7 +121,6 @@ extension PhotonActionSheetProtocol {
         }
         
         let addReadingList = PhotonActionSheetItem(title: Strings.AppMenuAddToReadingListTitleString, iconString: "addToReadingList") { action in
-            guard let tab = self.tabManager.selectedTab else { return }
```

Comment:
Huh. That's weird. I wonder why this was there since `tab` is already passed in?

Reply:
Yeah, I poked @farhanpatel about this — presumably an oversight or the result of a bad refactor.